<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>821</width>
      <height>541</height>
     </rect>
    </property>
    <property name="title">
     <string>GroupBox</string>
    </property>
    <widget class="QTabWidget" name="tabWidget">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>781</width>
       <height>491</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Gestion object</string>
      </attribute>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>Nom</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEditNom">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>40</y>
         <width>120</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>Categorie</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEditCategorie">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>70</y>
         <width>120</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>100</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>Brand</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEditBrand">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>100</y>
         <width>120</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>130</y>
         <width>90</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>Reference</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEditReference">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>130</y>
         <width>120</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>160</y>
         <width>100</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>Type de controle</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEditTypeDeControle">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>160</y>
         <width>120</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonAjouterObjet">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>190</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Ajouter</string>
       </property>
      </widget>
      <widget class="QTableView" name="tableViewObjet">
       <property name="geometry">
        <rect>
         <x>410</x>
         <y>80</y>
         <width>331</width>
         <height>191</height>
        </rect>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::AnyKeyPressed|QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed|QAbstractItemView::SelectedClicked</set>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEditIdObjet">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>10</y>
         <width>120</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>100</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>Id objet</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonSupprimerO">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>190</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Supprimer</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonModifierO">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>190</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>modifier</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_IdObjet">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>10</y>
         <width>70</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>Id objet</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_Nom">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>40</y>
         <width>70</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>Nom</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_Reference">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>130</y>
         <width>70</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>Reference</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_Brand">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>100</y>
         <width>70</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>Brand</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_Categorie">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>70</y>
         <width>70</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>Categorie</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_TypeDeControle">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>160</y>
         <width>101</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>type de controle</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_SortObject">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>190</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Sort</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBoxObjet">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>10</y>
         <width>62</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>ASC</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>DESC</string>
        </property>
       </item>
      </widget>
      <widget class="QLineEdit" name="lineEditRechercheO">
       <property name="geometry">
        <rect>
         <x>520</x>
         <y>10</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonRechercheO">
       <property name="geometry">
        <rect>
         <x>660</x>
         <y>10</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Rechercher</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Gestion de maintenance</string>
      </attribute>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>47</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Etat</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEditEtat">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>40</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>130</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Date </string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEditDate">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>130</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>100</y>
         <width>61</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Id reparteur</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEditReferenceM">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>70</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Reference</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEditIdReperateur">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>100</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonAjouterMaintenance">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>170</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Ajouter</string>
       </property>
      </widget>
      <widget class="QTableView" name="tableViewMaintenance">
       <property name="geometry">
        <rect>
         <x>420</x>
         <y>30</y>
         <width>256</width>
         <height>192</height>
        </rect>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::AnyKeyPressed|QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed|QAbstractItemView::SelectedClicked</set>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
      </widget>
      <widget class="QPushButton" name="SupprimerMaintenance">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>170</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>supprimer</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonM">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>170</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>modifier</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEditId_Maintenance">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>10</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_21">
       <property name="geometry">
        <rect>
         <x>6</x>
         <y>10</y>
         <width>91</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>ID MAINTENANCE</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_IdMaintenance">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>10</y>
         <width>101</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>Id Maintenance</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_Reference_2">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>70</y>
         <width>70</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>Reference</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_Date">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>130</y>
         <width>70</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>Date </string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_Etat">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>40</y>
         <width>70</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>Etat</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_IdReperateur">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>100</y>
         <width>81</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>Id reparteur</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_SortMaintenance">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>170</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Sort</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
